core/src/com/unciv/logic/battle/CityCombatant.kt

```diff
@@ -26,15 +27,15 @@ class CityCombatant(val city: CityInfo) : ICombatant {
     }
 
     override fun getUnitType(): UnitType = UnitType.City
-    override fun getAttackingStrength(): Int = getCityStrength()*2/5 // I remember reading this but I don't recall where
+    override fun getAttackingStrength(): Int = getCityStrength() //If we don't need to multiply by a modifier number (0.75) in Damage Calculations, getCityStrength() should multiply by 1/(x^i+0.5) (x = CityStrength / DefendUnitStrength, i = if(x>1) 1 else -1).
     override fun getDefendingStrength(): Int{
         if(isDefeated()) return 1
         return getCityStrength()
     }
 
     fun getCityStrength(): Int { // Civ fanatics forum, from a modder who went through the original code
         var strength = 8f
-        if(city.isCapital()) strength+=2.5f
+        if(city.isCapital()) strength+=2f
```

Comment:
Any reference?

Reply:
i watch movie, if there is a scout(5) in city, city's strength +1(it will be 8+2+5*0.2~~11), there is a warrior (8) in city, city's strength +2(it will be 8+2+8*0.2~~12), if we don't concern about other factor，it should +2f.

Reply:
Okay, I'm willing to accept that